query/graphql/src/main/java/org/codice/ddf/admin/graphql/transform/GraphQLTransformOutput.java

```diff
@@ -167,7 +170,7 @@ public Object functionDataFetcher(DataFetchingEnvironment env, FunctionField<Dat
         FunctionReport<DataType> result = funcField.getValue();
 
         if(!result.messages().isEmpty()) {
-            throw new FunctionDataFetcherException(funcField.fieldName(), args, result.messages());
+            throw new FunctionDataFetcherException(toPrettyString(funcField, result.messages()), result.messages());
```

Comment:
I think we should move the `toPrettyString` method and the ones it invokes into the `FunctionDataFetcherException` class. This should make it much easier to test that functionality.